vuepress/docs/v6.3.2/tutorials/samples/hub.md

```diff
@@ -0,0 +1,141 @@
+```
+```
+ent bundler from-git -r https://github.com/entando-samples/entando-hub-content-bundle.git -d | ent kubectl apply -n entando -f -
+```
+
+2. Log into your App Builder instance.
+
+3. Select `Repository` from the menu on the left. Your bundles will be visible in the repository as shown in the screenshot below.
+
+![hub-install.png](./hub-images/hub-install.png)
+
+4. Select `Install` for each bundle, where order of installation is important. The `entando-hub-bundle` will need to be installed first, as it provides the `entando-hub-content-bundle` with MFEs to set up each of the pages. An installation can take several minutes while the application downloads the Linux images for the microservices and installs the related assets.
+
+:::warning
+(Entando 6.3.2) There is a cache issue when deploying the Hub bundles where not all widgets or MFEs initially appear on some pages, particularly the Dashboard page. Restarting the quickstart-server pod (which holds the Entando App Engine) will clear the cache. This is only necessary on the initial install.
+:::
+
+5. The Hub can be accessed from the App Builder by navigating to `Pages → Management`, finding `Hub Test` **change** in the page tree, and clicking `View Published Page` from its actions.
+
+## Configuration
+
+::: tip
+(New with Entando 7.0.0) To point the Entando App Builder in another Entando Application to any Hub instance, configure the App Builder using the endpoint `BASEURL/entando-hub-api/appbuilder/api`, where the BASEURL is the URL for the Entando Application.
+:::
+
+## Using the Hub
+
+### Concepts
+
+The key entities in the Entando Hub are:
+
+- `Bundle Group`: A Bundle Group is a group of one or more Entando Bundles. 
+- `Bundle Group Version`: A Bundle Group can have one or more versions, each with a particular status.
+- `Bundle`: A Bundle is the deployment unit within an Entando Application. A Bundle can contain one or multiple components such as micro frontends, microservices, or any of the [component types](../../docs/ecr/ecr-bundle-details.html#overview) available in Entando. 
+- `Category`: Each Bundle Group belongs to a specific category. The initial possible categories are Solution Template, Packaged Business Capability (PBC), or Component Collection. An admin of an Entando Hub can refine the available categories as desired.
+- `Organization`: Bundle Groups belong to a single organization. Authors and managers can only update Bundle Groups for their own organization.  
+- `User`: User identity is managed within Keycloak, where users are granted roles within the Hub. Users must be assigned to a specific organization.
+
+Notes:
+- A private repository can be used for a Bundle, but this requires [an additional Kubernetes secret](../../docs/reference/identity-management.html#logging-into-your-keycloak-instance) before deployment via the App Builder.
```

Comment:
This is the correct link - http://localhost:8080/v6.3.2/tutorials/ecr/ecr-private-git-repo.html